src/components/Person.js

```diff
@@ -24,7 +24,7 @@ function useIntersectionObserver(ref) {
 
 export default function Person({ person, currentTag }) {
   const url = new URL(person.url);
-  const twitter = `https://unavatar.now.sh/twitter/${person.twitter.replace('@', '')}`;
```

Comment:
This string interpolation fails when it encounters a data entry with no `twitter` key in it—with the error `Cannot read property 'replace' of null`. My PR addresses it by a ternary operator to interpolate the string only when there's a `twitter` value.